Dockerfile

```diff
@@ -4,17 +4,17 @@ WORKDIR /go/src/github.com/openshift/hive
 COPY . .
 RUN make build
 
-FROM quay.io/centos/centos:7
+FROM quay.io/centos/centos:8
```

Comment:
What are the consequences of this as far as the whole CentOS Stream future is concerned?

Reply:
No idea what they're doing with the official centos images, possibly not a lot given that change is mostly about updates.